code/game/objects/items/devices/taperecorder.dm

```diff
@@ -185,19 +185,18 @@
 			break
 		if(mytape.storedinfo.len < i)
 			break
-		var/turf/T = get_turf(src)
-		T.visible_message("<font color=Maroon><B>Tape Recorder</B>: [mytape.storedinfo[i]]</font>")
+		atom_say("<font color=Maroon><B>Tape Recorder</B>: [mytape.storedinfo[i]]</font>")

```

Comment:
No formating, just:

`atom_say(mytape.storedinfo[i])` (be sure to test this though).